Gordon360/Services/ServiceInterfaces.cs

```diff
@@ -227,8 +227,8 @@ public interface IMembershipRequestService
     }
     public interface IScheduleService
     {
-        Task<IEnumerable<ScheduleViewModel>> GetScheduleStudentAsync(string username);
-        Task<IEnumerable<ScheduleViewModel>> GetScheduleFacultyAsync(string username);
+        Task<IEnumerable<ScheduleViewModel>> GetScheduleStudentAsync(string username, string? sessionID=null);
+        Task<IEnumerable<ScheduleViewModel>> GetScheduleFacultyAsync(string username, string? sessionID=null);
```

Comment:
```suggestion
        Task<IEnumerable<ScheduleViewModel>> GetScheduleStudentAsync(string username, string? sessionID = null);
        Task<IEnumerable<ScheduleViewModel>> GetScheduleFacultyAsync(string username, string? sessionID = null);
```
for styling purposes